plugins/pulp_docker/plugins/models.py

```diff
@@ -48,6 +50,16 @@ class Image(pulp_models.FileContentUnit):
             'indexes': [],
             'allow_inheritance': False}
 
+    def list_files(self):
+        """
+        List absolute paths to files associated with this unit.
+
+        :return: A list of absolute file paths.
+        :rtype: generator
+        """
+        for name in ('ancestry', 'json', 'layer'):
+            yield os.path.join(self.storage_path, name)
```

Comment:
I think there is no need to use a generator here since the list is only three items, but also there's no downside I suppose other than making the docblock language dance around whether it is a list or generator ☺ I'm rambling.